lib/phenix.rb

```diff
@@ -61,18 +64,20 @@ def for_each_database
     end
   end
 
-  def run_mysql_command(conf, command)
-    @mysql_command ||= begin
-      commands = [
-        'mysql',
-        "--user=#{conf['username']}"
-      ]
-      commands << "--host=#{conf['host']}" if conf['host'].present?
-      commands << "--port=#{conf['port']}" if conf['port'].present?
-      commands << " --password=#{conf['password']} 2> /dev/null" if conf['password'].present?
-      commands.join(' ')
-    end
-    `echo "#{command}" | #{@mysql_command}`
+  def run_mysql_command(conf, execute)
+    command = ['mysql']
+    command << "--user" << conf['username'].to_s if conf['username'].present?
```

Comment:
Wondering if `%w[user host port password].each { |param| command << param << conf[param].to_s if conf[param].present? }` is better or worse...